### PR TITLE
Feature/booster coverage

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard19**
Hold for BE to be ready
- [x] Ready

### CODE REVIEW

File: packages/app/schema/nl/booster_coverage.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    },
+    "last_value": {
+      "$ref": "#/definitions/value"
+    }
+  },
+  "required": ["values", "last_value"],
+  "additionalProperties": false,
+  "definitions": {
+    "value": {
+      "title": "nl_booster_coverage",
```
Comment (VWSCoronaDashboard20) [inline packages/app/schema/nl/booster_coverage.json:22]: The key has the same title as line 3, this results in interface names with a 1 behind them when generated based on this file. Are you sure this is correct?

File: packages/common/src/types/data.ts [hunk 2]
```diff
@@ -414,6 +414,15 @@ export interface NlBoosterShotPerAgeGroupValue {
   date_end_unix: number;
   birthyear_range: string;
 }
+export interface NlBoosterCoverage {
+  values: NlBoosterCoverage1[];
+  last_value: NlBoosterCoverage1;
+}
+export interface NlBoosterCoverage1 {
```
Comment (VWSCoronaDashboard20) [inline packages/common/src/types/data.ts:421]: The number 1 behind the interface usually indicates a duplicate key; see the previous comment.

**Review by VWSCoronaDashboard20 — CHANGES_REQUESTED** [review]

File: packages/app/schema/nl/booster_coverage.json (hunk 1 above)
Comment (VWSCoronaDashboard19) [reply packages/app/schema/nl/booster_coverage.json:22]: Sharp! that's correct, will double check it again and verify with be!

File: packages/app/schema/nl/booster_coverage.json [hunk 3]
```diff
@@ -0,0 +1,39 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "nl_booster_coverage",
+  "type": "object",
+  "properties": {
+    "values": {
+      "type": "array",
+      "minItems": 1,
+      "maxItems": 1,
```
Comment (VWSCoronaDashboard24) [inline packages/app/schema/nl/booster_coverage.json:9]: Is this correct? Max 1 value in the array?

Comment (VWSCoronaDashboard18) [reply packages/app/schema/nl/booster_coverage.json:9]: Yes, there is no historical data in this entry point

**Review by VWSCoronaDashboard18 — APPROVED** [review]

**Review by VWSCoronaDashboard24 — APPROVED** [review]

**Review by VWSCoronaDashboard22 — APPROVED** [review]

File: packages/app/src/pages/index.tsx [hunk 4]
```diff
@@ -559,7 +563,10 @@ const Home = (props: StaticProps<typeof getStaticProps>) => {
                               .booster_shots_administered_data_drive_text,
                             {
                               percentage: formatPercentage(
-                                boosterCoverageLastValue.percentage
+                                boosterCoverageFeature.isEnabled &&
```
Comment (VWSCoronaDashboard19) [inline packages/app/src/pages/index.tsx:566]: Added a feature flag here and also checking if the currently existing percentage is in `boosterCoverageEstimatedLastValue`. When `booster_coverage` data goes live, it won't show NaN on the dashboard, but switch over to the new data in the `else`

**Review by VWSCoronaDashboard19 — COMMENTED** [review]
Small note added

**Pull request merged**